drivers/display/ssd16xx.c

```diff
@@ -71,6 +71,12 @@ struct ssd16xx_config {
 	uint8_t tssv;
 	uint8_t pp_width_bits;
 	uint8_t pp_height_bits;
+
+	uint8_t dummy_line;
+	bool override_dummy_line;
+
+	uint8_t gate_line_width;
+	bool override_gate_line_width;
```

Comment:
I think it always depends on dummy line pulse setting, so override_gate_line_width is redundant.

Reply:
I think you are right. It doesn't make sense to specify one without specifying the other since they are dependant. However, I don't think it is possible to specify that relationship in the device tree, so I would like keep both as independent overrides.